docs/src/main/sphinx/connector/delta-lake.md

```diff
@@ -709,6 +709,14 @@ WITH (
 AS SELECT name, comment, regionkey FROM tpch.tiny.nation;
 ```
 
+(delta-lake-shallow-clone)=
+#### Shallow cloned tables
+
+The connector only supports reading from shallow cloned tables. The connector
```

Comment:
what about writing?

Reply:
Well.. is writing supported? We thought it is not

Reply:
I think we might end up having to add more detail on what is supported and what is not ... 

Here is what I think might be the case:

* creating shallow clone - not supported in trino, use spark 
* read on new table supported, but are there any restrictions?
* write on new table maybe supported, but what specifically happens if say you delete a row.. I think it would also delete the record on the source table...
* alter shallow clone clone - no idea
* Drop shallow clone table .. probably supported but needs confirmation

Reply:
Also lastly.. it would be good to have a least one example use case described that is a bit practical ..

Reply:
```
testing/bin/ptl env up --environment singlenode-delta-lake-oss
```

```
trino> create schema delta.tiny with (location='s3://test-bucket/tiny/');
CREATE SCHEMA
trino> create table delta.tiny.t1 (col integer);
CREATE TABLE
trino> insert into delta.tiny.t1 values 1, 2, 3;
INSERT: 3 rows

trino> insert into delta.tiny.t1 values 4, 5, 6;
INSERT: 3 rows
```

```
spark-sql (default)> create table tiny.t1clone shallow clone tiny.t1 ;
432	2	0	0	0	0
Time taken: 7.517 seconds, Fetched 1 row(s)
```

```
trino> select * from delta.tiny.t1clone;
 col 
-----
   4 
   1 
   2 
   3 
   5 
   6 
(6 rows)

trino> insert into delta.tiny.t1clone values 7, 8, 9;
INSERT: 3 rows


trino> select * from delta.tiny.t1clone;
 col 
-----
   7 
   8 
   9 
   4 
   5 
   6 
   1 
   2 
   3 
(9 rows)

trino> drop table delta.tiny.t1clone;
DROP TABLE
```

The above scenario showcases that both reading & writing are supported on current Trino version for Delta Lake cloned tables.

Probably there are some things (e.g. :  deletion vectors on shallow clones) that are not fully polished yet, but we're getting there. :)

Reply:
>  it would be good to have a least one example use case described that is a bit practical ..

One example use case is the ability to safely "experiment" on the cloned table without affecting the source table.

Reply:
Awesome ... so the only thing that doesnt work is the creation of the shallow cloned table it seems. 

@Joelg96 this should be enough info for you to update the PR.

Reply:
@mosabua made copy edits.